final-project/Figures/create-room-diag.mmd

```diff
@@ -0,0 +1,27 @@
+---
+title: Creating or Generating a Room
+---
+sequenceDiagram
+    actor Alice
+    participant OTT as OTT website
+    participant L as Load Balancer
+    participant N as Monolith
+    alt temporary room
+    Alice->>+OTT: Hello, I want to generate a temporary room
+    OTT->>+L: Serve new connection
+    Note over L: Determines which node has least load
+    L->>+N: "temp" is generated
```

Comment:
A uuid is generated and used as the room name when "generating". When "creating" the request must supply a room name. See `/api/room/generate/` vs `/api/room/create`. Basically "generate" means "make this thing for me" and "create" means "create this thing like i told you to".